### ISSUE TITLE
Dialogl, ok action is active even when hovering over cancel

### ISSUE THREAD

**Issue by omerts** [react-mdl/react-mdl#444]
Hey,

When the dialog opens, by default it sets the first button as active, as if it is hovered. If the other button is hovered, without going over the first button, the hovered state stays the same. 

You can see it in the demo: https://react-mdl.github.io/react-mdl/components/dialog/
Open the dialog, and hover over "disagree" without going over "agree", you will see "agree" staying active.

**Comment by kradio3** [comment]
This is not a bug of react-mdl. The same behavior reproduced on clean mdl see codepen http://codepen.io/kradio3/pen/dOqZQg

**Comment by omerts** [comment]
@kradio3 On the clean mdl hovering over the agree doesn't remove th accent from it

**Comment by kradio3** [comment]
@omerts Yes, i found it. That because of .mdl-js-button. http://codepen.io/kradio3/pen/rWqeqB

#### Button doesn't lose focus on mouseleave
```html
<button class="mdl-button">Agree</button>
```

#### Button loses focus on mouseleave
```html
<button class="mdl-button mdl-js-button">Agree</button>
```

Because of losing focus keyboard's Enter will have different behaviors

react-mdl includes mdl-js-button class by default
https://github.com/react-mdl/react-mdl/blob/master/src/Button.js#L27

This .mdl-js-button required by getmdl.io https://getmdl.io/components/index.html#buttons-section
